api/webscraper/utils/scraper_utils.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import json
+import pandas as pd
+from io import BytesIO
+import urllib
+from dotenv import load_dotenv
+import os
+
+load_dotenv('.env.local')
+
+google_maps_api_key = os.environ.get('NEXT_PUBLIC_GOOGLE_MAPS_API_KEY')
+
+def check_status(status):
+    if status is None:
+        return 'Proposed'
+    elif status.lower() == 'cancelled':
+        return 'Cancelled'
+    elif status.lower() == 'operational' or status.lower() == 'completed':
+        return 'Operational'
+    else:
```

Comment:
let's return proposed only if status says under development

Reply:
got it! what should we return if the status is missing? I can leave it as a `null`/`None` status and then add error handling for that in the other scraper files.

Reply:
let's leave status as `None` if it is missing